ModelLoaderException : /opt/ros/hydro/share/nextage_description/models/main.wrl cannot be found.

On new installed machine, `ModelLoaderException` repeats in a very fast loop.

```
$ roslaunch nextage_ros_bridge nextage_ros_bridge.launch nameserver:=nextage &> roslaunch_bridge.log
:
ModelLoaderException : /opt/ros/hydro/share/nextage_description/models/main.wrl cannot be found.
[HrpsysJointTrajectoryBridge] failed to load model[]
ModelLoaderException : /opt/ros/hydro/share/nextage_description/models/main.wrl cannot be found.
:
ModelLoaderException : /opt/ros/hydro/share/nextage_description/models/main.wrl cannot be found.
[ERROR] [1404979055.697524972]: Target Node visual1/node_joint0_axis0 NOT found!!!
[rtmlaunch] starting...  /opt/ros/hydro/share/hrpsys_ros_bridge/launch/hrpsys_ros_bridge.launch
ModelLoaderException : /opt/ros/hydro/share/nextage_description/models/main.wrl cannot be found.
ModelLoaderException : /opt/ros/hydro/share/nextage_description/models/main.wrl cannot be found.
[rtmlaunch] RTCTREE_NAMESERVERS nextage:15005 nextage:15005
[rtmlaunch] SIMULATOR_NAME RobotHardware0
[rtmlaunch] check connection/activation
:
[ INFO] [1404979056.045774554]: [HrpsysSeqStateROSBridge] @Initilize name : HrpsysSeqStateROSBridge0

SUMMARY
========

PARAMETERS
 * /collision_state/comp_name
 * /controller_configuration
 * /diagnostic_aggregator/analyzers/computers/contains
 * /diagnostic_aggregator/analyzers/computers/path
 * /diagnostic_aggregator/analyzers/computers/type
 * /diagnostic_aggregator/analyzers/hrpsys/contains
 * /diagnostic_aggregator/analyzers/hrpsys/path
 * /diagnostic_aggregator/analyzers/hrpsys/type
 * /diagnostic_aggregator/analyzers/mode/contains
 * /diagnostic_aggregator/analyzers/mode/path
 * /diagnostic_aggregator/analyzers/mode/type
 * /diagnostic_aggregator/analyzers/motor/contains
 * /diagnostic_aggregator/analyzers/motor/path
 * /diagnostic_aggregator/analyzers/motor/type
 * /diagnostic_aggregator/base_path
 * /diagnostic_aggregator/pub_rate
 * /robot_description
 * /rosdistro
 * /rosversion

NODES
  /
    CollisionDetectorComp (hrpsys/CollisionDetectorComp)
    DataLoggerServiceROSBridge (hrpsys_ros_bridge/DataLoggerServiceROSBridgeComp)
:

$ ls -lath /opt/ros/hydro/share/nextage_description/models/main.wrl
-rw-r--r-- 1 root root 25K Jun 20 19:10 /opt/ros/hydro/share/nextage_description/models/main.wrl
```

```
$ dpkg -p ros-hydro-nextage-description|grep Ver
Version: 0.2.14-0precise-20140621-0805-+0000
$ dpkg -p ros-hydro-nextage-ros-bridge|grep Ver
Version: 0.2.14-0precise-20140623-0906-+0000
$ dpkg -p ros-hydro-hironx-ros-bridge|grep Ver
Version: 1.0.16-1precise-20140708-0336-+0000
$ dpkg -p ros-hydro-openhrp3|grep Ver
Version: 3.1.5-6precise-20140617-0805-+0000
$ dpkg -p ros-hydro-hrpsys|grep Ver
Version: 315.2.2-0precise-20140617-1729-+0000
```

--- a/nextage_ros_bridge/launch/nextage_ros_bridge_real.launch
+++ b/nextage_ros_bridge/launch/nextage_ros_bridge_real.launch
@@ -1,0 +1,17 @@
+<launch>
+  <arg name="CONF_FILE_COLLISIONDETECT" default="$(find hironx_ros_bridge)/conf/hironx_realrobot_fixedpath.conf" />
+  <arg name="corbaport" default="15005" />
+  <arg name="MODEL_FILE" default="/opt/jsk/etc/HIRONX/model/main.wrl" /> <!-- This shouldn't be changed unless you know what you're doing; this is the location of vrml file internal to the robot (on QNX OS). -->
+  <arg name="nameserver" default="nextage" /> <!-- Host name of the QNX. -->
+  <arg name="USE_SERVOCONTROLLER" default="false" />
+
+  <env name="RTCTREE_NAMESERVERS" value="$(arg nameserver):$(arg corbaport)" />
+
+  <include file="$(find hironx_ros_bridge)/launch/hironx_ros_bridge_real.launch" >
+    <arg name="CONF_FILE_COLLISIONDETECT" value="$(arg CONF_FILE_COLLISIONDETECT)" />
+    <arg name="corbaport" value="$(arg corbaport)" />
+    <arg name="MODEL_FILE" value="$(arg MODEL_FILE)" />
+    <arg name="nameserver" value="$(arg nameserver)" />
+    <arg name="USE_SERVOCONTROLLER" value="$(arg USE_SERVOCONTROLLER)" />
+  </include>
+</launch>

